index.html

```diff
@@ -168,33 +183,27 @@ <h3>MediaDevices.produceCropTarget</h3>
           </dt>
           <dd>
             <p>
-              The first call of {{MediaDevices/produceCropTarget}} on an {{HTMLElement}} of a
-              supported type, permanently associates that element with a {{CropTarget}}. Subsequent
-              calls return the same {{CropTarget}}. This {{CropTarget}} may be used as input to
+              Calling {{MediaDevices/produceCropTarget}} on an {{HTMLElement}} of a
+              supported type associates that element with a {{CropTarget}}.
+              This {{CropTarget}} may be used as input to
               {{BrowserCaptureMediaStreamTrack/cropTo}}. We define a
               <dfn>valid CropTarget</dfn> as one returned by a previous call to
               {{MediaDevices/produceCropTarget()}} in the current [=browsing context=].
             </p>
             <p>
-              When {{MediaDevices/produceCropTarget}} is first called on a given <var>element</var>,
-              the user agent MUST produce a new unique {{CropTarget}} and permanently associate it
-              with <var>element</var>. The user agent MUST return a {{Promise}} <var>P</var>. The
+              When {{MediaDevices/produceCropTarget}} is called on a given <var>element</var>,
+              the user agent [=create a CropTarget|creates a CropTarget=] with <var>element</var> as input.
+              The user agent MUST return a {{Promise}} <var>P</var>. The
               user agent MUST resolve <var>P</var> only after it has finished all the necessary
               internal propagation of state associated with the new {{CropTarget}}, at which point
               the user agent MUST be ready to receive the new {{CropTarget}} as a valid parameter to
               {{BrowserCaptureMediaStreamTrack/cropTo}}.
             </p>
-            <p>
-              Subsequent calls to {{MediaDevices/produceCropTarget}} MUST return a new {{Promise}}
-              that behaves the same way <var>P</var> does. Specifically, the new {{Promise}} MUST
-              resolve after <var>P</var>, and MUST resolve to the same {{CropTarget}} as
-              <var>P</var>.
-            </p>
             <p>
               When cloning an {{HTMLElement}} on which {{MediaDevices/produceCropTarget}} was
-              previously called, the clone MUST NOT be associated with any {{CropTarget}}. If
+              previously called, the clone is not associated with any {{CropTarget}}. If
               {{MediaDevices/produceCropTarget}} is later called on the clone, a new {{CropTarget}}
-              MUST be assigned to it.
+              will be assigned to it.
```

Comment:
To me, this seems related to @youennf's change above to produce a new cropTarget every time, which makes this normative prose redundant. We could remove it entirely as well, since a cloned element is clearly a new element, so I don't see why we need to say anything.

Reply:
Yes, I made it non normative since it is no longer needed. I would be for removing it entirely.

Reply:
> @youennf's change above to produce a new cropTarget every time

I did not understand this change as requiring a new CropTarget each time, but rather as permitting it.

Reply:
> @youennf's change above to produce a new cropTarget every time

I did not understand this change as requiring a new CropTarget each time, but rather as permitting it.